deps/spidershim/test/value.cc

```diff
@@ -1870,6 +1870,94 @@ TEST(SpiderShim, FunctionCall) {
   EXPECT_TRUE(r10->StrictEquals(True(isolate)));
 }
 
+TEST(SpiderShim, ConstructCall) {
+  V8Engine engine;
+
+  Isolate::Scope isolate_scope(engine.isolate());
+
+  HandleScope handle_scope(engine.isolate());
+  Local<Context> context = Context::New(engine.isolate());
+  Context::Scope context_scope(context);
+  Isolate* isolate = engine.isolate();
+
+  engine.CompileRun(context,
+      "function Foo() {"
+      "  var result = [];"
+      "  for (var i = 0; i < arguments.length; i++) {"
+      "    result.push(arguments[i]);"
+      "  }"
+      "  return result;"
+      "}");
+  Local<Function> Foo = Local<Function>::Cast(
+      context->Global()->Get(context, v8_str("Foo")).ToLocalChecked());
+
+  v8::Local<Value>* args0 = NULL;
```

Comment:
Can you please remove these needless `v8::` prefixes?  I find the tests much easier to read without this spread all over them.  :-)